WooCommerce/Classes/ViewRelated/Products/Edit Product/ProductFormViewModelProtocol.swift

```diff
@@ -137,6 +140,10 @@ protocol ProductFormViewModelProtocol {
 
     func updateVariationAttributes(_ attributes: [ProductVariationAttribute])
 
+    func checkIfBlazeIntroViewIsNeeded()
+
+    func updateShouldShowBlazeIntroView(to newValue: Bool)
```

Comment:
As commented above, these are redundant and can be removed.

Reply:
resolved in https://github.com/woocommerce/woocommerce-ios/pull/11011/commits/7cc08fde5ad0a04b3be82451b7473f1468d99409